lib/matplotlib/pyplot.py

```diff
@@ -1131,106 +1131,11 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
         # same as
         plt.subplots(2, 2, sharex=True, sharey=True)
     """
```

Comment:
Not super worried about the duplication of docstrings at this point.  An interesting follow on to this PR would be to generate `plt.subplots()` via `boilerplate.py`.